learn-arai/app/globals.css

```diff
@@ -92,3 +92,24 @@ label {
     font-weight: 700;
     font-size: 16;
 }
+
+input {
+    color: black;
+    border: solid #d1d5db 3px;
+    border-radius: 8px;
+    padding: 8px;
+    text-indent: 26px;
+}
+
+input:focus {
+    border: solid #9ba7b6 3px;
+    outline: none; /*remove default outline browser border*/
+}
+
+.icon-in-input-field {
+    position: absolute;
+    top: 50%;
+    transform: translate(0, -50%);
+    fill: #d1d5db;
+    left: 12px;
+}
```

Comment:
Why are you global stylign input, this will change every input on our web any particular reasons?

Reply:
Good question....but why I do that maybe boobo did that
![image](https://github.com/learn-arai/learn-arai/assets/116646446/828d7070-127f-4a93-91ed-7c62264f538b)
Just a some dog mistake I will fix